kobo-uncaged/util.go

```diff
@@ -74,3 +76,21 @@ func readJSON(fn string, out interface{}) (emptyOrNotExist bool, err error) {
 
 	return false, json.NewDecoder(f).Decode(out)
 }
+
+func wrapPos(err error) error {
+	if err != nil {
+		pc, file, line, ok := runtime.Caller(1)
+		if !ok {
+			return wrap(err, "[unknown pos]")
+		}
+		return wrap(err, "[0x%X %s:%d]", pc, file, line)
+	}
+	return nil
+}
+
+func wrap(err error, format string, a ...interface{}) error {
```

Comment:
Why wrap an error instead of fixing it?
```suggestion
func fix(err error) error {
    return nil
}
```